cluster/gce/util.sh

```diff
@@ -477,7 +477,7 @@ function make-gcloud-network-argument() {
     ret="${ret},aliases=pods-default:${alias_size}"
     ret="${ret} --no-can-ip-forward"
   else
-    ret="--network ${network}"
+    ret="--subnet=${network}"
```

Comment:
Should we check if the users network is not legacy? Does this work with the legacy network?